pennylane/operation.py

```diff
@@ -582,6 +582,7 @@ def decompose(self):
             return self.decomposition(wires=self.wires)
         return self.decomposition(*self.parameters, wires=self.wires)
 
+    # pylint: disable=unused-argument
     @staticmethod
     def compute_diagonalizing_gates(*params, wires, **hyperparams):
```

Comment:
I think that putting the disabling above the matrix might disable it at the class level?

I've always done the following instead

```python
def method(self, args):  # pylint: disable=...
```

or 


```python
def method(self, args):
    # pylint: disable=...
```

I think this is better, as another developer adding a separate method with issues might then be skipped by pylint?

Reply:
Oh I see, will change!